packages/client/bin/utils.ts

```diff
@@ -495,7 +495,7 @@ async function setupDevnet(prefundAddress: Address, args: ClientOpts) {
             epoch: 30000,
           },
         }
-  const defaultChainData = {
+  const defaultChainData: Omit<GethGenesis, 'alloc'> = {
```

Comment:
Nit: I think `alloc` is mandatory in Geth, would it be easier here to just do `alloc: {}` for empty alloc by default?